### ISSUE TITLE
Opportunities and challenges

### ISSUE THREAD

**Issue by delapuente** [w3c/webvr-content-workshop#38]
Dear hosts of the follow-up sessions,

Here you have a space to identify opportunities and challenges. Please leave a comment with your conclusions about the session you hosted. Try to get it complete before tomorrow, December, the 7th at 11.00 am.

Thank you!

**Comment by Utopiah** [comment]
Landscape of WebVR authoring tools : 
- Opportunity : Extension system for each platform editing tool focusing on their vertical based on the components they are the most efficient for  
- Challenge : Standardizing for those extensions across those different platforms

Other opportunities : 
- make (Github) issues of each framework and their tooling friendlier for newcomers to efficiently suggest new and important features, not solely bug
- efficiently adding high quality (low poly or not) 3D assets
- simplifying interaction for non developers
- in VR editing for non developers

Other challenges : 
- bringing more developers to WebVR by promoting immersive content on platform e.g Facebook with 360 photos rising to the top
-  avoiding the curse of knowledge by keeping in mind newcomers might have no VR, 3D or programming experience
- in VR editing for non developers without requiring high-end hardware (i.e roomscale and 6DoF)

**Comment by delapuente** [comment]
You can write more than one if you want. ;) Let's say, *up to three*.

**Comment by Aadjou** [comment]
**Creating, packaging, delivering 3D/360° assets for webVR**
- Challenge: Create and identify standards for 3D assets / cubemaps (eg. origin of a model, standard scale (+ unit size), coordinate system, face sides, ...)
- Opportunity: Enabling to apply those standards with debugging tool (eg. interactive helper that helps to fix problems with your model)
- Opportunity: Identify use-cases for ZIP-like file packaging (offline application support, shared security options ...)

**Comment by billyco** [comment]
Encouraging diversity in VR development

- Challange: Currently the industry is dominated by white male people which make it unwelcoming to more people from diverse groups. The dominant group might not even be aware of bad practices they are doing.
- Opportunity: We can encourage diversity by hosting/ organizing exclusive events that would give ways for diverse people to learn more about the industry and get involved in the industry through a welcoming environment. This would balance the number of people of different color, sex, background, religion, etc. and would finally lead to the lack of need for exclusive events.
- Opportunity: We can all commit and put efforts in building more diverse events with speakers representing different groups as this usually brings more people from this group to attend the events.

**Comment by learnbrite** [comment]
Social VR

- Challenge: Building social vr demo's is easy, building robust (social and technical) vr is a lot of work

- Opportunity: A set of standards and components to address the common issues that appear in any synchronous multiuser experience. Its the responsibility of the site owner, and being able to quickly choose which components or standards to add to a site could accelerate this aspect of VR.

**Comment by ghost** [comment]
Use/business cases and commercialisation of WebVR 

- Challenge: To be enable to push compeling experiences on the actually (performance) limited devices
- Challenge: To bring people toward VR when the use of it is not clear as only 5% of them click on the "Enter VR" button from a WebVR experience (Vizor statistics)

 - Opportunity: bring clients and users to the VR market by taking them by the hand and showing them proper use-cases and showcasing them as much as possible

**Comment by ghost** [comment]
Progressive enhancement and user interaction applied to the variety of user input in WebVR 
- Challenge: offer accessibility to experiences through every type of device (be it a desktop with mouse/keyboard or a 6dof headset with controllers)
- Challenge: create an easy-to-use interface for users to interact with the content in a proper way

- Opportunity: push browser vendors to create a powerful IK system enabling to mimic a human's skeleton motion based on any type of input available (enabling full range of movement through a skeleton controled by a mouse/3dof controler/6dof controler...)
- Opportunity: give the users a single settings interface through the browser enabling them to have the same configuration to interact throughout experiences

**Comment by mkungla** [comment]
**Spatial audio**

- Challenge: it's still difficult to create really complex and rich audio scenes for WebXR (e.g using many audio tracks or swapping audio sources )
- Opportunity:  We need to collect use cases and ideas to see how to advance the conversation  and bring momentum in improving audio experience in WebXR
- Opportunity: Connecting audio instruments in WebXR is another topic which should be covered. 

---

- Challenge: Identify lack of the right primitives in the Web Audio API or create standards for spatial / ambisonic sound field based on POW without sacrificing the integrity of sound,
- Opportunity: Give content creators ways to add the acoustics and spatial feel to the audio based on POW

---

- Challenge: Identify resources, networking, WebRTC, user-gesture requirement related performance bottlenecks and synchronization challenges.
- Opportunity: Deliver low latency sound within Social WebXR experiences

---

- Challenge: Identify how spatial sound can help to deliver Accessible XR experience.

**Comment by Oletus** [comment]
Performance

**Challenge**: It's difficult to determine what the limits on an end-user device are in terms of memory capacity or rendering throughput.

**Opportunity**: Better developer tools for analyzing performance, especially GPU performance.

**Opportunity**: Simulating lower-end devices could make developing for a variety of hardware easier.

**Comment by isabelcatalansada** [comment]
VR Storytelling
These are the challenges and Opportunities we discussed:


Opportunities:
•	

WebVR Storytelling 
o	3D, 360degree, and Interaction
o	There are NO Creative and Distribution Limitations
•	Engaging the VR-loving Users and Audience
o	Emotion Association to VR Experience (Use of Visual, Auditive Keys)
o	Multi-choice/Linear story-based VR Experiences
o	Multiple user-Cases for VR in different sectors for different audiences/viewers
•	Transmedia
o	WebVR and VR becoming part of other Media sectors
o	WebVR and VR accessible to all Industries and sectors
o	WebVR and VR becoming a new creative/educational technology

Challenges:
•	WebVR Storytelling Challenges:
o	Music: finding new ways to implement music/other artforms to create compelling WebVR and VR experiences. 
o	Artforms: Finding the technology to Map, Position, Compose, Time and Situate the users, and create the artistic environment in the WebVR and VR Experiences. (Tools)
o	How do we create the key factors (Control, Failure and Surprise) for the multi-choice and Linear story-based WebVR and VR experiences. (Tools for creating control, failure and surprise inside the experience).
•	Engaging the VR-loving Users and Audiences:
o	Creating tools to bridge the gap between content creators and the tech-developers. 
o	Making collaborative efforts that envolve the content creators and tech developer (Techies & Creatives uniting)
o	Finding our Audiences (Who are they?) And Keeping them in mind when we create tools and experiences
o	USING Users to develop better experiences and tools.
o	Raising awareness for the technology and the content available to all potential users and viewers. (Educate the users, and bridge the gap between techs & creatives)
o	Creating “code-free” online tools for users to learn about the available technologies and learn about WebVR and VR (XR, AR).
•	FINANCE:
o	How to raising awareness on the importance of VR as a Financially sustainable tool/experience
o	How do we find ways to Monetize the content and tools available. (Merch, Marketing, etc…)
o	Finding the possible use of Crypto currencies, Blockchain and WebPayment as a form to sustain the experiences and monetize them. (Tokens, WebPayment etc…)

**Comment by AVGP** [comment]
Regarding the 3D assets, I think we should move forward and create something like the "Web 3D Creators Guidelines" or something, based on the idea from @anialdam. 

It would give a reference for 3D asset creators, e. g. "Y is up", "1 unit = 1 meter", "center around origin" (those are just examples and not necessarily the recommendations). 

How would we best get get that off the ground?

**Comment by mkungla** [comment]
@AVGP I think for now best would be to open pull request to this repository `./proposals/3D-assets.md` and elaborate follow up from there

**Comment by delapuente** [comment]
I think a dedicated page is a better place for this kind of info. What is the format you're suggesting @AVGP ? A static page with guidelines?

**Comment by AVGP** [comment]
Yes, pretty much. I think it'd be great to create a repo somewhere under
the w3c organisation and then start rounding up a bunch of people who are
interested in creating guidelines.

2017-12-19 17:27 GMT+01:00 Salvador de la Puente González <
notifications@github.com>:
